templates/shared/issuelist.tmpl

```diff
@@ -59,6 +59,29 @@
 					{{else}}
 						{{$.i18n.Tr .GetLastEventLabelFake $timeStr (.Poster.GetDisplayName | Escape) | Safe}}
 					{{end}}
+					{{if and .IsPull}}
+						<div class="branches">
+							<div class="branch">
+								<a href="/{{if .PullRequest.BaseRepo}}{{.PullRequest.BaseRepo.OwnerName}}/{{.PullRequest.BaseRepo.LowerName}}{{end}}/src/branch/{{.PullRequest.BaseBranch}}">
```

Comment:
Incorrect URL here.

Reply:
Could you give more details pls? You want to change URL generation from frontend to backend and substitute URL like
`<a class="index ml-0 mr-2" href="{{if .HTMLURL}}{{.HTMLURL}}{{else}}{{$.Link}}/{{.Index}}{{end}}">` ?

Reply:
Gitea may run under a sub-path, eg: `https://mydomain.com/gitea/org/repo`

So, constructing an absolute URL from root will be incorrect in such case.

Reply:
Fixed, add {{$.AppSubURL}} to URLs